From 403f7ba479e64532d0afecc120e77a7cf6380dd4 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Tue, 2 Jul 2019 21:01:15 -0300
Subject: [PATCH] systemd-networkd-wait-online.service.in: use --any by default

Use --any by default when waiting for a network interface to be fully
configured, otherwise it blocks until all the available interfaces are
in the configured state.

Upstream-Status: Inappropriate [lmp specific]

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 units/systemd-networkd-wait-online.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-networkd-wait-online.service.in b/units/systemd-networkd-wait-online.service.in
index 7666e16669..0d4f09ef1e 100644
--- a/units/systemd-networkd-wait-online.service.in
+++ b/units/systemd-networkd-wait-online.service.in
@@ -18,7 +18,7 @@ Before=network-online.target shutdown.target
 
 [Service]
 Type=oneshot
-ExecStart=@rootlibexecdir@/systemd-networkd-wait-online
+ExecStart=@rootlibexecdir@/systemd-networkd-wait-online --any
 RemainAfterExit=yes
 
 [Install]
-- 
2.17.1

